dj_database_url.py

```diff
@@ -7,6 +7,11 @@
 except ImportError:
     import urllib.parse as urlparse
 
+try:
+    from django import VERSION as DJANGO_VERSION
+except ImportError:
+    DJANGO_VERSION = None
```

Comment:
When will this be None?

Reply:
When Django is not installed. There's no installation requirements in `setup.py`, and I didn't want to make the module installable but not importable.
...although adding an installation requirement for Django is probably the better way, huh?